understand/observed-size-spectra.qmd

```diff
@@ -219,13 +220,11 @@ ggplot(binned_numbers) +
     geom_line(aes(x = mid_points[bin], y = Number_dens)) + 
     scale_x_log10() + 
     scale_y_log10() +
-    labs(x = "Log10 Weight [g]",
```

Comment:
The values given on the axes are the weight and the number density, not their logs. These values are simply displayed on a logarithmic scale.